internal/saucecloud/imagerunner.go

```diff
@@ -158,6 +158,7 @@ func (r *ImgRunner) runSuite(suite imagerunner.Suite) (imagerunner.Runner, error
 	var run imagerunner.Runner
 	metadata := make(map[string]string)
 	metadata["name"] = suite.Name
+	metadata["resourceProfile"] = suite.ResourceProfile
```

Comment:
Waiting for confirmation from Dan.

Reply:
It should work. Otherwise I got 
```
11:19:33 ERR Suite finished. error="runner start failed (400): {\"code\":\"ERR_CREATE_RUNNER_BAD_REQ_BODY_INVALID_VALUE\",\"message\":\"We where not able to process Request Body, because requested resource profile is not supported. Expected one of: [c1m1 c2m2 c3m3], got: test\"}\n" passed=false runID= suite="Playwright in ImageRunner"
```